README.md

```diff
@@ -0,0 +1,144 @@
+    - [🔭 Future-feature ](#-future-feature-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Country Details Project <a name="about-project"></a>
+
+**Country Details ** allows users to search for countries and check the details of searched country
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>React</li>
+    <li>Redux</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://github.com/">Github</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Search Countries**
+- **Select Countries**
+- **Check Country Details**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy and run, follow these steps.
+### Prerequisites
+Visual Studio Code
+Chrome Browser
+Node.js
+### Setup
+1- Clone this repository to your desired folder
+
+2- cd my-folder
+
+3- git clone
+
+```
+  https://github.com/kamranahmad123/Country-Details
+```
+
+4- cd country-Details
+
+### Install
+npm install
+### Usage
+npm run build
+npm start
+
+<!-- AUTHORS -->
+
+## 👥 Authors 
+<a name="authors"></a>
+
+👤 **Kamran Ahmad**
+__Kamran Ahmad__
+- GitHub: [@githubhandle](https://github.com/kamranahmad123) 
+- Twitter: [@twitterhandle](https://twitter.com/handel)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kamran-khan-9a293b113/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future-feature <a name="future-features"></a>
+
+- [ ] **Display and fetch more country data in the future**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+ We would like to thank all our fellow micronauts.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+- **What is this project?**
+
+  - It's a  project of micoverse curriculum entitled Country Details.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/kamranahmad123/Country-Details/blob/main/LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# leaderboard
```

Comment:
Great job @kamranahmad123 🤝

- [ ] Kindy delete this line consisting of `leaderboard` to make your readme file more professional.

Reply:
- [ ] Well resolved 🥇